### PR TITLE
Makeshift Tools

### PR DESCRIPTION
**Pull request by Afya212**
## About The Pull Request

Adds makeshift tools, inferior versions of their standard counterparts.

## Why It's Good For The Game

Allows poor quality tools to be built if you have no other option.
"Tools only being available from youtools doesn't make sense considering we can make entire laserguns out've garbage"

## Changelog
:cl: Afya
add: Added makeshift tools
imageadd: Makeshift tool sprites courtesy of JungleRat
/:cl:

Tools
image

### CODE REVIEW

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 1]
```diff
@@ -20,3 +20,66 @@
...
+	tools = list(/obj/item/hammer/makeshift)
+	result = /obj/item/screwdriver/makeshift
+	reqs = list(/obj/item/stack/cable_coil = 2,
+				/obj/item/stack/rods = 2)
+	time = 80
+	subcategory = CAT_TOOL
+	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/welder
+	name = "Makeshift Screwdriver"
```
Comment (BobJoga) [inline modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:55]: uhhhhhhhhh

**Comment by BobJoga** [comment]
Love this PR, but there is quite a few things i find lacking which you could probably port from this PR on hippie https://github.com/HippieStation/HippieStation/pull/11127

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm (hunk 1 above)
Comment (Afya212) [reply modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:55]: AAAAAAAAA

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 2]
```diff
@@ -20,3 +20,66 @@
 	time = 40
 	subcategory = CAT_TOOL
 	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/crowbar
+	name = "Makeshift Crowbar"
```
Comment (jjpark-kb) [inline modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:25]:
```suggestion
	name = "makeshift crowbar"
```
consistency

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 3]
```diff
@@ -20,3 +20,66 @@
...
+	tools = list(/obj/item/hammer/makeshift)
+	result = /obj/item/crowbar/makeshift
+	reqs = list(/obj/item/stack/sheet/metal = 4,
+				/obj/item/stack/cable_coil = 2)
+	time = 120
+	subcategory = CAT_TOOL
+	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/hammer
+	name = "Makeshift Hammer"
```
Comment (jjpark-kb) [inline modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:35]:
```suggestion
	name = "makeshift hammer"
```
consistency

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm [hunk 4]
```diff
@@ -20,3 +20,66 @@
...
+	result = /obj/item/hammer/makeshift
+	reqs = list(/obj/item/stack/cable_coil = 2,
+				/obj/item/stack/rods = 2,
+				/obj/item/kitchen/rollingpin = 1)
+	time = 80
+	subcategory = CAT_TOOL
+	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/screwdriver
+	name = "Makeshift Screwdriver"
```
Comment (jjpark-kb) [inline modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:45]:
```suggestion
	name = "makeshift screwdriver"
```
consistency

File: modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm (hunk 2 above)
Comment (jjpark-kb) [reply modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm:25]: im an idiot, this is recipes. Disregard all of these changes to recipe name changes.

**Pull request merged**